### PR TITLE
Add a couple missing fields

### PR DESCRIPTION
**Pull request by tejasmanohar**
I spotted that a couple fields were missing so added them. They're all in https://stripe.com/docs/api

### CODE REVIEW

File: transfer.go [hunk 1]
```diff
@@ -71,29 +71,30 @@ type TransferListParams struct {
...
+	Meta               map[string]string   `json:"metadata"`
+	Bank               *BankAccount        `json:"bank_account"`
+	Card               *Card               `json:"card"`
+	Recipient          *Recipient          `json:"recipient"`
+	Statement          string              `json:"statement_descriptor"`
+	Reversals          *ReversalList       `json:"reversals"`
+	Reversed           bool                `json:"reversed"`
+	SourceTx           *TransactionSource  `json:"source_transaction"`
+	SourceType         TransferSourceType  `json:"source_type"`
+	DestinationPayment string              `json:"destination_payment"`
```
Comment (brandur) [inline transfer.go]: Thanks! Can you rename this one to `DestinationPayment`? The naming choices in this library kill me, but I think it's a good idea to stay consistent with other struct fields.

Comment (tejasmanohar) [inline transfer.go]: @brandur isn't it called `DestinationPayment`?

Comment (tejasmanohar) [inline transfer.go]: ahh, fixed.

**Comment by brandur** [comment]
LGTM.

Thanks!

**Comment by brandur** [comment]
Released as 18.2.0.

**Pull request merged**